src/data-mgt/node/utils/errors.js

```diff
@@ -1,56 +1,28 @@
 const HTTPStatus = require("http-status");
+const { logText, logObject, logElement } = require("./log");
 
-const axiosError = ({ error = {}, res = {}, extra = [] } = {}) => {
-  let errors = extra;
-  if (error.response) {
-    // that falls out of the range of 2xx
-    res.status(HTTPStatus.BAD_GATEWAY).json({
+const errors = {
+  convertErrorArrayToObject: (controller) => {
+    const initialValue = {};
+    return controller.reduce((obj, item) => {
+      let param = item.param;
+      return {
+        ...obj,
+        [param]: `${item.msg}`,
+      };
+    }, initialValue);
+  },
+  errorResponse: ({ res, message, statusCode = 500, error = {} } = {}) => {
+    logObject("error", error);
```

Comment:
To emphasise the global error handling, merged all the functions into one for all error responses irrespective of the type